src/Plugin/GraphQL/Fields/Entity/Fields/View/ViewDerivative.php

```diff
@@ -62,7 +62,7 @@ public function resolveValues($value, array $args, ResolveContext $context, Reso
     $view = EntityView::load($values['target_id']);
     $display = $this->getViewDisplay($view, $values['display_id']);
     $this->pluginDefinition['paged'] = $this->isPaged($display);
-    $this->pluginDefinition['argument_info'] = $this->getArgumentsInfo($display->getOption('arguments') ?: []);
```

Comment:
Hahaha yes i was just going to comment this, that instead of my patch, just this line will suffice.